client/state/terms/test/actions.js

```diff
@@ -56,10 +56,10 @@ const taxonomyName = 'jetpack-testimonials';
 const categoryTaxonomyName = 'category';
 
 describe( 'actions', () => {
-	const spy = sinon.spy();
+	let spy;
 
 	beforeEach( () => {
-		spy.resetHistory();
+		spy = sinon.spy();
```

Comment:
apparently resetHistory doesn't let go of everything? Making a new spy on every run fixed a leak warning. Not sure it's valid though.

Reply:
Let's just rip out sinon and chai. Jest can do it all!

#28907

Reply:
thanks!